tracer/src/Datadog.Trace/RuntimeMetrics/PssRuntimeInformation.cs

```diff
@@ -0,0 +1,226 @@
+// <copyright file="PssRuntimeInformation.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+#nullable enable
+
+using System;
+using System.ComponentModel;
+using System.Runtime.InteropServices;
+using Datadog.Trace.Logging;
+
+namespace Datadog.Trace.RuntimeMetrics;
+
+// ReSharper disable InconsistentNaming UnusedMember.Local
+internal class PssRuntimeInformation
```

Comment:
Super small nit: Can we expand `Pss` to `ProcessSnapshot` in the class name, so `ProcessSnapshotRuntimeInformation`?

Reply:
Done in https://github.com/DataDog/dd-trace-dotnet/pull/5304/commits/0fe295ca7c0793caa050f01d8a9e3211e76e452b